frontend/src/components/homePage/HomePage.jsx

```diff
@@ -27,13 +26,9 @@ function App() {
     ]
 
     useEffect(() => {
-        if (changeInEvents) {
-            setIsLoading(true)
-            fetchEvents()
-            fetchEventsWithCategory(currentCategory)
-        }
+        fetchEvents()
+        fetchEventsWithCategory(currentCategory)
         setIsLoading(false)
-        setChangeInEvents(false)
```

Comment:
Same as above. When do you actually want to re-fetch events from db? Every time events state changed? But what does your fetchEvents change? Isn't it eventState?